spec/src/main/asciidoc/configuration.adoc

```diff
@@ -0,0 +1,307 @@
+//
+// Copyright (c) 2023 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[sec:configuration-properties]]
+== Configuration
+
+OpenTelemetry MUST be configured by MicroProfile Config following the semantics of configuration properties detailed in https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}/sdk-extensions/autoconfigure[OpenTelemetry SDK Autoconfigure {otel-java-version}].
+Following properties MUST be supported:
+
+=== Configuration Properties
+// Table with property name, default value and description
```

Comment:
We should mention the date and version of the properties bellow, since we are copying the content of the OpenTelemetry SDK Autoconfigure page.

Reply:
We haven't done it when SDK autoconfigure  was in alpha to break away from spec-level alpha dependency.

It is stable now, but it is still a subset of properties.

Reply:
I prefer not to list all of the configurations but rather mention the ones this spec does not require as we pretty much support all of them.

Reply:
I agree with @Emily-Jiang